frontend/src/components/7.Result/ResultPage.js

```diff
@@ -110,41 +123,171 @@ function ResultPage(props) {
+                        }}
+                      >
+                        See results
+                      </Button>
+
+                      <Result trigger={ResultPopup} setTrigger={setResultPopup}>
+                        {chart && (
+                          <DataVisual className="DataVisual" data={pie} />
+                        )}
+                        <hr />
+                      </Result>
+                    </Col>
+
+                    <Col
+                      style={{
+                        display: "flex",
+                        alignItems: "center",
+                        justifyContent: "center",
+                      }}
+                      lg={12}
+                    >
+                      <Button
+                        onClick={() => setMapPopup(true)}
+                        className="BigBtn"
+                        id="GoogleMaps_btn"
+                        style={{ marginTop: "0.15em", marginBottom: "0.15em" }}
+                      >
+                        View on Google Maps
+                        <Icon />
+                      </Button>
+                      <MapModal
+                        trigger={MapPopup}
+                        setTrigger={setMapPopup}
+                        restaurantLocation={{ lat: -36.8523, lng: 174.7691 }}
+                      />
+                    </Col>
+                  </Row>
+                </Card.Body>
+              </>
+            ) : (
+              <>
+                <Card.Header
+                  as="h5"
+                  id="Card-Header"
+                  className="text-center"
+                  style={{ width: "100%" }}
+                >
+                  No Results Decided!
+                </Card.Header>
+                <Card.Body
+                  style={{
+                    display: "flex",
+                    alignItems: "center",
+                    justifyContent: "center",
+                  }}
+                >
+                  <Row>
+                    <Col
+                      style={{
+                        display: "flex",
+                        alignItems: "center",
+                        justifyContent: "center",
+                      }}
+                      lg={12}
+                    >
+                      <h2 className="ResultTitle"> Want to try again? </h2>
+                    </Col>
+                    <Col
+                      style={{
+                        display: "flex",
+                        alignItems: "center",
+                        justifyContent: "center",
+                      }}
+                      lg={12}
+                    >
+                      <Button
+                        className="ButtonReset"
+                        onClick={() => window.location.reload()}
+                      >
+                        CONFIRM{" "}
+                      </Button>
+                    </Col>
+                  </Row>
+                </Card.Body>
```

Comment:
Could you please replace all of this with:
```
                <Card.Header
                  as='h5'
                  id='Card-Header'
                  className='text-center'
                  style={{ width: '100%' }}
                >
                  No Result Decided!
                </Card.Header>
                <Card.Body className='text-center'>
                  <p>Your group was not keen on any restaurants.</p>
                  <Button
                    variant='danger'
                    onClick={() => window.location.reload()}
                  >
                    Try Again{' '}
                  </Button>
                </Card.Body>
```

Reply:
Done and it looks really clean. Nice changes Reeve